src/pudl/transform/ferc1.py

```diff
@@ -267,6 +295,29 @@ class Ferc1AbstractTableTransformer(AbstractTableTransformer):
     the transformed data.
     """
 
+    xbrl_metadata_json: list[dict[Any]] = []
+    """An array of JSON objects extracted from the FERC 1 XBRL taxonomy."""
+
+    xbrl_metadata_normalized: pd.DataFrame = pd.DataFrame()
+    """A semi-normalized dataframe containing table-specific XBRL metadata."""
+
+    def __init__(
+        self,
+        params: TableTransformParams | None = None,
+        cache_dfs: bool = False,
+        clear_cached_dfs: bool = True,
+        ferc1_xbrl_raw_meta: list[dict[Any]] | None = None,
+    ) -> None:
+        """Augment inherited initializer to store XBRL metadata in the class."""
+        super().__init__(
+            params=params,
+            cache_dfs=cache_dfs,
+            clear_cached_dfs=clear_cached_dfs,
+        )
+        # Many tables don't require this input:
+        if ferc1_xbrl_raw_meta:
+            self.xbrl_metadata_json = ferc1_xbrl_raw_meta
```

Comment:
i don't love that these are named differently. is there a reasons for this or can we standardize them?

Reply:
I standardized on `xbrl_metadata_json` and applied that change everywhere.